<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPatientDialog</class>
 <widget class="QDialog" name="NewPatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Patient Records</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../startuptoolresources.qrc">
    <normaloff>:/assets/icons/_48_start_document.png</normaloff>:/assets/icons/_48_start_document.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(172, 172, 172);
font: 16pt &quot;Bell MT&quot;;
</string>
  </property>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 22pt &quot;Bell MT&quot;;</string>
   </property>
   <property name="text">
    <string>Patient Information</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="procedureGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>681</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Procedures</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>661</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>131</width>
      <height>87</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBtn">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>511</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QComboBox" name="bodyPartListComboBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="procedureListComboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>770</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>770</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>421</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>21</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="lastNameLbl">
      <property name="text">
       <string>Last Name*</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lastNameLineEdit">
      <property name="toolTip">
       <string extracomment="patient last name"/>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>421</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="firstNameLbl">
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="firstNameLineEdit">
      <property name="toolTip">
       <string extracomment="patient first name"/>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>421</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="middleNameLbl">
      <property name="text">
       <string>Middle Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="middleNameLineEdit">
      <property name="toolTip">
       <string extracomment="middle name"/>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>421</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>25</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="patientIDLbl">
      <property name="text">
       <string>Patient ID*</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="patientIdLineEdit">
      <property name="toolTip">
       <string extracomment="patient ID"/>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>254</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="birthDateLbl">
      <property name="text">
       <string>Birth Date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>214</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="GenderLbl">
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="genderComboBox">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unknown</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>453</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="refessingPhysicianLbl">
      <property name="text">
       <string>Referring Physician</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="referringPhysicianLineEdit">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>451</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="admissionNumberLbl">
      <property name="text">
       <string>Admission Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="admissionNumberLineEdit">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>453</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="accessionNumberLbl">
      <property name="text">
       <string>Accession Number*</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="accessionNumberLineEdit">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../startuptoolresources.qrc"/>
 </resources>
 <connections/>
</ui>
